streaming/text/convert/c4.py

```diff
@@ -0,0 +1,109 @@
+import os
+from argparse import ArgumentParser, Namespace
+from typing import Any, Dict, Iterator
+
+import datasets
+from datasets.arrow_dataset import Dataset
+from torch.utils.data import DataLoader, IterableDataset, get_worker_info
+from tqdm import tqdm
+
+from ...base import MDSWriter
+
+
+def parse_args() -> Namespace:
+    """Parse commandline arguments.
+
+    Args:
+        Namespace: Commandline arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument('--out', type=str, default='/datasets/mds/c4-zstd/')
+    args.add_argument('--compression', type=str, default='zstd:7')
```

Comment:
Same as above.

Reply:
Eh?

Reply:
My bad :D . My comment was for `args.add_argument('--out', type=str, default='/datasets/mds/c4-zstd/')` to have a default path from current dir rather than root. Based on your [this](https://github.com/mosaicml/streaming/pull/4/#discussion_r967383343) comment, we are anyhow leaning towards making a required argument so ignore this comment.